rumbl/priv/repo/backend_seeds.exs

```diff
@@ -0,0 +1,4 @@
+alias Rumbl.Repo
+alias Rumbl.User
```

Comment:
I have found `alias Rumbl.{Repo, User}` to be quite a nice pattern, but I'd say not worth changing